cdap-data-fabric/src/test/java/co/cask/cdap/data2/registry/UsageRegistryTest.java

```diff
@@ -16,20 +16,43 @@
 
 package co.cask.cdap.data2.registry;
 
+import co.cask.cdap.data2.dataset2.DatasetFrameworkTestUtil;
+import co.cask.cdap.data2.nosql.NoSqlStructuredTableAdmin;
+import co.cask.cdap.data2.nosql.NoSqlStructuredTableRegistry;
+import co.cask.cdap.data2.nosql.NoSqlTransactionRunner;
+import co.cask.cdap.spi.data.StructuredTableAdmin;
+import co.cask.cdap.spi.data.TableAlreadyExistsException;
+import co.cask.cdap.store.StoreDefinition;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 import org.apache.tephra.TransactionSystemClient;
 import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 
-public class UsageRegistryTest extends UsageDatasetTest {
+import java.io.IOException;
+
+public class UsageRegistryTest extends UsageTableTest {
```

Comment:
Does this class need to extend from UsageTableTest? Since it is using a `NoSqlTransactionRunner`, all the functionalities should already be tested in `NoSqlUsageTableTest`.